client/src/main.rs

```diff
@@ -159,13 +167,28 @@ pub async fn main() -> Result<(), Box<dyn Error>> {
         )
         .get_matches();
 
-    let _address = matches.value_of("address").unwrap();
+    let address = matches.value_of("address").unwrap();
+    let mut client = proto::Client::new(String::from(address).try_into().unwrap()).await.map_err(|err| err.compat())?;
 
     if let Some(_matches) = matches.subcommand_matches("ping") {
-        unimplemented!();
+        client.ping().await.map_err(|err| err.compat())?;
+        println!("ok");
     } else if let Some(matches) = matches.subcommand_matches("set") {
         if let Some(_matches) = matches.subcommand_matches("vertex") {
-            unimplemented!();
+            let link_type = indradb::Type::new(_matches.value_of("type").unwrap()).map_err(|err| err.compat())?;
+            let uuid = match _matches.value_of("id") {
+                Some(id) => uuid::Uuid::parse_str(id)?,
+                None => indradb::util::generate_uuid_v1(),
+            };
+            let vertex = indradb::Vertex::with_id(uuid, link_type);
+            let res = client.transaction()
+                .await.map_err(|err| err.compat())?
+                .create_vertex(&vertex)
+                .await.map_err(|err| err.compat())?;
+            if res == false {
+                panic!("Failed to create vertex with id: {} and type: {}", vertex.id, vertex.t.0);
+            }
+            println!("Created vertex with id: {} and type: {}", vertex.id, vertex.t.0);
```

Comment:
Maybe just print the debug version of the vertex (i.e., `println!("{:?}", vertex);`), that way it should be easier to parse if this is used in a script